### DISCUSSION TITLE
WAV Files - No "AlbumArtist" tag?

### DISCUSSION THREAD

**Discussion by Emilot** [jcorporation/myMPD#1296] [Q&A] [answered]
myMPD version: [15.0.1]

I have an album with .wav files and it is a compilation (a lot of Artist tags).
I read somewhere that .wav files do not have "AlbumArtist" tag and therefore in myMPD i see a lot of "boxes" with Artist and AlbumArtist the same name, even i have Various Artists tag on AlbumArtist.

It is true that about .wav files?

Reproduce

Try to add a compilation album with .wav files.

How can we manage these compilation albums without AlbumArtist tag, in the case of .wav files, to diplay them like the rest (flac, etc)?

![εικόνα](https://github.com/jcorporation/myMPD/assets/18086896/e5a3a36f-9928-4895-98a6-fa8e4c7b1626)

No errors of course at logs.

**Comment by jcorporation** [comment]
That's a MPD issue. myMPD does not read any tag values directly, it get's all tags from MPD.

**Reply by Emilot** [reply]
Hmmm... Yes i know it gets all the tags from MPD, but it is MPD issue or client thing? MPD can read the tags but myMPD cannot diplay it right under one album label, right? 

Is my thought wrong? And this is an exception only to .wav files (i hate this format).

**Reply by jcorporation** [reply]
myMPD makes no distinction between file formats. myMPD simply uses the tags from MPD. If your files taged with the AlbumArtist tag and MPD reads this tag and provides this tag to the client, grouping should work. For myMPD there is no difference by file format. You should look if the response from MPD contains the AlbumArtist tag.

**Reply by Emilot** [reply, marked as answer]
As i understand MPD has nothing to do with it. It is the format that sucks. WAV cannot have AlbumArtist tag.

myMPD cannot display it under one album because it needs AlbumArtist. So, no luck, i guess.

I've just want to make it clear and to clarify if i get it right.

**Reply by jcorporation** [reply]
I think that is correct

**Comment by Emilot** [comment]
Just to clear this issue:

If you have ffmpeg decoder enabled at MPD and you have this issue with AlbumArtist tag in WAV files, then you must disable ffmpeg(at mpd.conf file). 
After deactivation the tag is showing and myMPD displays it correctly.

PS. I haven't an idea if you can disable ffmpeg only for wav files in the mpd.conf file.

**Reply by jcorporation** [reply]
Good to know, it seems the ffmpeg plugin have some limitations of reading tags.
